en/concepts/scene/light.md

```diff
@@ -1,40 +1,17 @@
 # Lighting
 
-The __Light__ in the game represents an object with luminous ability that can illuminate the surrounding environment. Adding light in the scene can make the scene produce the corresponding light and shadow effects, and get better visual effects.
+This chapter introduces how lighting works and how to use it in Cocos Creator.
```

Comment:
```suggestion
This document introduces how lighting works and how to use it in Cocos Creator.
```

Reply:
Actually it is the whole chapter not just this document introduces how lighting works and how to use it in Cocos Creator.

Reply:
ok, I see that. However, we don't designate chapters very much.

<img width="306" alt="Screen Shot 2021-09-28 at 18 01 25" src="https://user-images.githubusercontent.com/7491758/135171793-668cf531-12cd-4a70-ab44-4d6dbe23079f.png">

If we standardize on the use of `chapter` we should describe the other sections as chapters. It seems we have been using `document` and `documentation`.